plugins/catalog-import/src/api/CatalogImportClient.ts

```diff
@@ -40,12 +48,17 @@ export class CatalogImportClient implements CatalogImportApi {
   }: {
     repo: string;
   }): Promise<PartialEntity[]> {
+    const idToken = await this.identityApi.getIdToken();
+    const headers = {
+      'Content-Type': 'application/json',
+    } as { [header: string]: string };
+    if (idToken) {
+      headers.authorization = `Bearer ${idToken}`;
```

Comment:
``authorization: idToken && `Bearer ${idToken}` `` in the object literal is perhaps a bit nicer here and below 😅